x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_nodejs/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "id": "apm_nodejs",
+  "title": "APM: NodeJS",
+  "description": "Detect anomalies in transactions, spans, and errors",
```

Comment:
Detect abnormal traces, anomalous spans, and identify periods of decreased throughput.

Reply:
Fixed in https://github.com/elastic/kibana/pull/53792/commits/144c0ba1c62d4da9d1fc35a03d930a1537bfb205